Lombiq.NodeJs.Extensions/Lombiq.NodeJs.Extensions.targets

```diff
@@ -30,36 +34,90 @@
   <!-- Try to enable pnpm via corepack, which is part of Node.js v16.9 and above. -->
   <Target Name="EnablePnpm">
     <Exec Command="corepack enable" IgnoreExitCode="true" />
+    <!-- Enable the latest version of pnpm@6. -->
+    <Exec Command="corepack prepare pnpm@6.32.24 --activate" IgnoreExitCode="true" />
+  </Target>
+
+  <!-- Update all relevant Node.js Extensions files in the consumer project. -->
+  <Target Name="UpdateNodeJsExtensionsNpmPackage" BeforeTargets="Clean"
+          DependsOnTargets="DeleteNodeJsExtensionsNpmPackage;AddNodeJsExtensionsAsNpmPackage">
+  </Target>
+
+  <!-- Delete all Node.js Extensions files in the consumer project. -->
+  <Target Name="DeleteNodeJsExtensionsNpmPackage">
+    <!-- Remove previous version of the Node.js Extensions npm package; keeping node_modules in place, though, for
+         performance reasons. -->
+    <ItemGroup>
+      <NodeJsExtensionsNpmPackageFoldersToDelete Condition="!Exists($(NodeJsExtensionsNpmPackageSourcePath))" />
+      <NodeJsExtensionsNpmPackageFoldersToDelete
+        Include="$([System.IO.Directory]::GetDirectories($(NodeJsExtensionsNpmPackageSourcePath)))"
+        Exclude="$([System.IO.Directory]::GetDirectories($(NodeJsExtensionsNpmPackageSourcePath), 'node_modules'))"
+        Condition="Exists($(NodeJsExtensionsNpmPackageSourcePath))" />
+    </ItemGroup>
+    <RemoveDir Directories="@(NodeJsExtensionsNpmPackageFoldersToDelete)" ContinueOnError="true" />
```

Comment:
Re `ContinueOnError` see this convo, might be applicable: https://github.com/Lombiq/NPM-Targets/pull/20#discussion_r915075950